src/collections/_documentation/platforms/native/index.md

```diff
@@ -431,7 +437,11 @@ sentry_init(options);
 
 The crashpad handler executable must be shipped alongside your application so
 that it can be launched when initializing the SDK. The path is evaluated
-relative to the current working directory at runtime.
+according to shell lookup rules at runtime.
+
+It is advised that users of the Native SDK specify both the path to the
```

Comment:
_nit_: Since this document is talking to the "users", we could phrase this more directly:

```suggestion
It is advised to specify both the path to the
```